library/src/main/java/com/novoda/downloadmanager/NotificationChannelCreator.java

```diff
@@ -0,0 +1,30 @@
+package com.novoda.downloadmanager;
+
+import android.app.NotificationChannel;
+import android.app.NotificationManager;
+import android.content.Context;
+import android.os.Build;
+
+public class NotificationChannelCreator {
+
+    public static String createDownloadNotificationChannel(Context context) {
+        if (CHANNEL_ID == null) {
+            CHANNEL_ID = "download-manager";
+            createNotificationChannelForAndroidOreo(context);
```

Comment:
Using a `static` to ensure that the setting of the channel is only done once per instance of the application, we always want to supply the basic channel identifier because it is used to create the notification.